config/pipeline.yaml

```diff
@@ -895,6 +895,30 @@ jobs:
       job_timeout: 10
     kcidb_test_suite: kselftest.cpufreq
 
+  kselftest-cpufreq-suspend: &kselftest-job
```

Comment:
No, please make use of the YAML anchors instead of redefining them (see the definition for `kselftest-dmabuf-heaps` just below). Your new definitions should have only the `collections` and `env` params explicitly defined, everything else should come from anchors.

Reply:
@a-wai thanks for reviewing it, I just force-pushed the changes as you requested.

Reply:
Thanks, looks good now